src/functions.js

```diff
@@ -66,12 +66,14 @@ function filter(text) {
       {pattern: '\\&', newValue: '&'},
       {pattern: '\\#', newValue: '#'},
       {pattern: '\\_', newValue: '_'},
+      {pattern: '\\texteuro', newValue: '€'},
       {pattern: /(\\author{)(.*?)(})/g, newValue: '$2'},
       {pattern: /(\\begin{)(.*?)(})(\[.*?])?/g, newValue: ''},
       {pattern: /(\\bibliography{)(.*?)(})/g, newValue: ''},
       {pattern: /(\\bibliographystyle{)(.*?)(})/g, newValue: ''},
-      {pattern: /(\\caption{)(.*?)(})/g, newValue: ''},
```

Comment:
As mentioned in the commit :

> Capture the trailing } when there is an \function{} being used in inside the \caption{}. 
> Example: \caption{ A small caption \cite{wtv} }

The in \cite{wtv} last } would be assumed as the last } of the \caption leaving a } in the textarea ... Therefore the change I propose captures the missing }.

Reply:
This will only work when the cite is at the end of the line. `\caption{ A small caption \cite{wtv} is not long}` will fail, resulting in `A small caption \cite{wtv is not long}`